### PR TITLE
Permission fixes on daemon, install script clarifications, and instant create-val 

### PR DESCRIPTION
**Pull request by Daniel-VDM**
Fixed the permission error that was brought up by P-OPS.
Added installation clarifications as well as allow sudo to install with a warning.  
Will now create a validator instantly and deactivate until node synced, then will activate.

### CODE REVIEW

**Review by fxfactorial — APPROVED** [review]

**Comment by fxfactorial** [comment]
what's the upgrade process anyway for auto_node?

**Comment by Daniel-VDM** [comment]
> what's the upgrade process anyway for auto_node?

Just rerun the install script

File: AutoNode/daemon.py [hunk 1]
```diff
@@ -24,6 +26,9 @@
     start as start_monitor,
     ResetNode
 )
+from.util import (
```
Comment (rhazberries) [inline AutoNode/daemon.py]: Add a space

**Review by rhazberries — APPROVED** [review]

**Comment by fxfactorial** [comment]
Will auto node let me know if there is a newer version of itself available ?

**Pull request merged**